server/api.js

```diff
@@ -3,8 +3,33 @@ import db from "./db.js";
 const router = Router();
 
 router.get("/videos", async (_, res) => {
-	const result = await db.query("SELECT * FROM videos;");
-	res.json(result.rows);
-});
+	try{
+		const result = await db.query("SELECT * FROM videos;");
+		res.json(result.rows);
+	}
+	catch(error){
+		console.error("Database connection error:", error);
+		res.status(500).json({success:false, error: "Could not connect to the database"});
+	}
+}); 
+
+router.delete("/api/videos/:id", async (req, res) => {
```

Comment:
Because of you already pointed to the path `/api/` in app.mjs, if you create endpoint with `/api/...` , you will need to go `/api/api/...` to use the endpoint. Could you change the endpoint to `/videos/:id` ? This way you will create the endpoint `/api(from app.mjs)/videos/:id(from api.js)`

Reply:
ohh yeahh yeah makes sense, thank you. I have changed it accordingly.